### PR TITLE
forms data

### PR DESCRIPTION
**Pull request by vennela413**
getting inputs

### CODE REVIEW

**Comment by su66uu** [comment]
Please use proper names for PR titles. The title should tell what the PR contians

File: forms.html [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+              I agree to the <a href="#">terms and conditions</a>
+            </label>
+          </div>
+        </div>               
+      </div>
+    </div>
+  <br><br>
+        <div class="field">
+          <p class="control">
+            <button class="button is-success" id="submit" onclick="myfunction()">
```
Comment (su66uu) [inline forms.html:102]: If you developing in Jquery, use jquery everywhere. Here `onclick` is javascript event. There is no change in output if you use onclick. But it is not good to use jquery in some places and javascript in other places

File: forms.html [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        </div> 
+      </form>
+      <div class="card">
+        <div class="card-content">
+      <p id="name"></p>
+      <p id="Email"></p>
+      <p id="Number"></p>
+      <p id="Text"></p>
+      <p id="gender"></p>
+      </div>
```
Comment (su66uu) [inline forms.html]: Here arrangement not even.

File: forms.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    $("#number").val("");
+    var text= $("#textarea").val();
+    $("#Text").html(text);
+    $("#textarea").val("");
+    var radioValue = $("input[name='member']:checked").val();
+   if(radioValue){
+    $("#gender").html( radioValue);
+     }
+   $(".radioButton").prop('checked',false);
+    }
```
Comment (su66uu) [inline forms.js]: Here also arrangement not even

**Review by su66uu — CHANGES_REQUESTED** [review]

**Comment by su66uu** [comment]
Post your all questions here. Like which part is not working. Where do you need help? What did you try? etc.

File: forms.html (hunk 2 above)
Comment (vennela413) [reply forms.html]: where i do make mistake anna,i use this for card display ,what is mistake in it

File: forms.js (hunk 3 above)
Comment (vennela413) [reply forms.js]: oka naa

File: forms.html (hunk 1 above)
Comment (vennela413) [reply forms.html:102]: but click event in jquery not working for display values,how can i get values

**Comment by vennela413** [comment]
the  question i want to ask how to rest the form ,
form

**Comment by vennela413** [comment]
anna i use reset method ,but i can't get certain result

File: forms.html (hunk 1 above)
Comment (su66uu) [reply forms.html:102]: Find out. I'm sure there is way to do that.

File: forms.html (hunk 2 above)
Comment (su66uu) [reply forms.html]: Nothing mistake here. The arrangement is missing.

**Comment by su66uu** [comment]
> anna i use reset method ,but i can't get certain result

No issues for now. We will sort it out. Resolve the above comment firstly. We will make the form work later

File: forms.html (hunk 2 above)
Comment (vennela413) [reply forms.html]: ohh okk anna

**Pull request closed without merging**